pkg/gadgets/trace/dns/types/dns.go

```diff
@@ -35,6 +35,7 @@ type Event struct {
 	PktType    string     `json:"pktType,omitempty" column:"type,minWidth:7,maxWidth:9"`
 	QType      string     `json:"qtype,omitempty" column:"qtype,minWidth:5,maxWidth:10"`
 	DNSName    string     `json:"name,omitempty" column:"name,width:30"`
+	Timestamp  uint64     `json:"timestamp,omitempty" column:"timestamp"`
```

Comment:
I wasn't sure what other properties to set for `column` here. I see that [traceloop sets `width:7,fixed,hide`](https://github.com/inspektor-gadget/inspektor-gadget/blob/e3546b642cefcbb5df31b11e3592219c0118facd/pkg/gadgets/traceloop/types/types.go#L34), but that truncated the timestamps. Thoughts?